### PR TITLE
130 adding asterick in required input field namephone and dob and removing default showing of choose option from the gender input field

### LINKED ISSUE #130: Adding  asterick(*) in required input field -name,phone and dob  and removing default showing of choose option from the gender input field

### PR DESCRIPTION
**Pull request by amirthapamagar2053**

### CODE REVIEW

File: src/sections/form/index.js [hunk 1]
```diff
@@ -9,18 +9,22 @@ export const defaultValues = {
...
-  startDate: null,
-  endDate: null,
-  password: "",
-  confirmPassword: "",
-  editor: "",
-  photo: null,
-  terms: false,
+  // age: "",
+  bloodGroup: "",
+  // startDate: null,
```
Comment (rushilshakya) [inline src/sections/form/index.js]: remove commented code

File: src/sections/form/index.js [hunk 2]
```diff
@@ -9,18 +9,22 @@ export const defaultValues = {
...
+  // confirmPassword: "",
+  // editor: "",
+  gender: "",
+  // photo: null,
+  // terms: false,
 };
 
 const MAX_FILE_SIZE = 2 * 1000 * 1000; // 2 Mb
 const FILE_FORMATS = ["image/jpg", "image/jpeg", "image/gif", "image/png"];
+const bloodGroups = ["A+", "A-", "B+", "B-", "AB+", "AB-", "O+", "O-"];
```
Comment (rushilshakya) [inline src/sections/form/index.js]: put these values in config

File: src/sections/form/index.js [hunk 3]
```diff
@@ -9,18 +9,22 @@ export const defaultValues = {
...
+const genders = ["male", "female", "other"];
```
Comment (rushilshakya) [inline src/sections/form/index.js]: put in config file

File: src/sections/form/index.js [hunk 4]
```diff
@@ -31,38 +35,40 @@ export const FormSchema = Yup.object().shape({
   email: Yup.string()
     .required("Email is required")
     .email("That is not an email"),
-  age: Yup.number()
-    .required("Age is required")
-    .positive("Age must be a positive number")
-    .integer()
-    .moreThan(18, "Age must be greater than or equal to 18")
-    .lessThan(120, "Age must be less than or equal to 120"),
+  // age: Yup.number()
```
Comment (rushilshakya) [inline src/sections/form/index.js]: remove commented out code

**Review by rushilshakya — CHANGES_REQUESTED** [review]

**Review by iamsuraj01 — APPROVED** [review]

**Review by rushilshakya — APPROVED** [review]

**Pull request merged**